libraries/stdlib/genosl/lib/mx_transform_color.osl

```diff
@@ -0,0 +1,13 @@
+color mx_srgb_texture_to_lin_rec709(color inColor)
+{
+    color breakPnt = color(0.03928571566939354, 0.03928571566939354, 0.03928571566939354);
+    color slope = color(0.07738015800714493, 0.07738015800714493, 0.07738015800714493);
+    color scale = color(0.9478672742843628, 0.9478672742843628, 0.9478672742843628);
+    color offset = color(0.05213269963860512, 0.05213269963860512, 0.05213269963860512);
+    color isAboveBreak = color(outColor.rgb[0] > breakPnt.rgb[0] ? 1.0 : 0.0,
```

Comment:
Syntax error here: outColor should be inColor :)

Reply:
Thanks for the catch, @bernardkwok!